tests/radiation_transport/test_radiation_profile.py

```diff
@@ -144,6 +154,13 @@ def test_calculate_mp_radiation_profile(self):
         assert len(self.source.core_rad.rad_mp) == 4
         assert rad_tot[0] > rad_tot[-1]
 
+    def test_calculate_core_distribution(self):
+        # calls calculate_core_distribution() internally
+        self.source.core_rad.calculate_core_radiation_map()
+        assert np.isclose(np.sum(self.source.core_rad.rad_tot), 273.55834367261707)
+        assert np.isclose(np.sum(self.source.core_rad.x_tot), 55614.05333940474)
+        assert np.isclose(np.sum(self.source.core_rad.z_tot), 239.84335614864912)
```

Comment:
Less room for flakey tests, it might need a little adjustment
```suggestion
        assert np.sum(self.source.core_rad.rad_tot) == pytest.approx(273.5583)
        assert np.sum(self.source.core_rad.x_tot) == pytest.approx(55614.0533)
        assert np.sum(self.source.core_rad.z_tot) == pytest.approx(239.84336)
```